spec/linter-rubocop-spec.js

```diff
@@ -77,43 +75,19 @@ describe('The RuboCop provider for Linter', () => {
     });
 
     it('verifies the first message', () => {
-      const msgText = 'Prefer single-quoted strings when you don&#39;t need ' +
-        'string interpolation or special symbols. ' +
-        '(<a href="https://github.com/bbatsov/ruby-style-guide#consistent-string-literals">Style/StringLiterals</a>)';
-
-      waitsForPromise(() =>
-        lint(editor).then((messages) => {
-          expect(messages[0].type).toEqual('Warning');
-          expect(messages[0].html).toBe(msgText);
-          expect(messages[0].text).not.toBeDefined();
-          expect(messages[0].filePath).toEqual(invalidWithUrlPath);
-          expect(messages[0].range).toEqual([[1, 6], [1, 20]]);
-        }),
-      );
-    });
-  });
-
-  describe('shows errors without a clickable link in a file with warnings', () => {
-    let editor = null;
-
-    beforeEach(() => {
-      waitsForPromise(() =>
-        atom.workspace.open(invalidWithoutUrlPath).then((openEditor) => { editor = openEditor; }),
-      );
-    });
-
-    it('verifies the first message', () => {
-      const msgText = 'Extra empty line detected at class body end. ' +
-        '(Style/EmptyLinesAroundClassBody)';
+      const msgText = 'Prefer single-quoted strings when you don\'t need ' +
```

Comment:
I usually switch to `"` strings so you don't need to escape the `'`, not a necessary change though.